docs/develop/api_for_vms.rst

```diff
@@ -0,0 +1,46 @@
+Api to communicate with VMS
+===========================
+The purpose of building this API was to provide information about the meetups
+in Portal to VMS so that their volunteers could participate in them. Presently,
+there are three fields of meetups which are sent to VMS via the API::
+
+1. Event Name
+2. Start Date
+3. Venue
+
+Users can send in a ``GET`` or a ``POST`` request to the specified `URL <localhost/meetup/api/v1/request_meetup_data/>`_ to access the meetup data.
```

Comment:
On clicking the `URL` it shows a 404 page. 
Remove `URL` and instead replace it with https://localhost/meetup/api/v1/request_meetup_data/